exporters/committime/collector_image.py

```diff
@@ -137,7 +137,7 @@ def _set_commit_time_from_annotations(
                 COMMIT_DATE_ANNOTATION_ENV,
                 CommitMetric._ANNOTATION_MAPPIG.get("commit_time"),
             )
-            commit_time = metric.annotations.get(commit_time_annotation)
+            commit_time = metric.annotations.get(commit_time_annotation).strip()
```

Comment:
```
            commit_time = metric.annotations.get(commit_time_annotation)
            if commit_time:
                commit_time = commit_time.strip()            
                metric.commit_time = commit_time
```

Reply:
Actually logged https://github.com/konveyor/pelorus/issues/620 but I now see it's part of your PR not the master :)

Reply:
or even:
```
metric.commit_time = commit_time.strip()
```

Reply:
thanks! done